### ISSUE TITLE
https/domain hosting

### ISSUE THREAD

**Issue by damienen** [bogdan-rosianu/elrond-converters#2]
Hi! Could this be hosted under a domain name/using https in order to increase community trust in the tool?

Thank you!

**Comment by newbreedofgeek** [comment]
Yes, that would be great @bogdan-rosianu. Maybe via Github Pages as it will be linked direct to Github : https://docs.github.com/en/pages/quickstart
